apps/ui/src/views/Space/Proposals.vue

```diff
@@ -115,6 +150,71 @@ watchEffect(() => setTitle(`Proposals - ${props.space.name}`));
             }
           ]"
         />
+        <div v-if="space.labels?.length" class="sm:relative">
+          <PickerLabel
+            v-model="labels"
+            :labels="space.labels"
+            :button-props="{
+              class: [
+                'flex items-center gap-2 relative rounded-full leading-[100%] border button h-[42px] top-1 text-skin-link bg-skin-bg',
+                {
+                  'max-w-[230px]': labels.length >= MAX_SHOWN_LABELS_FILTER
```

Comment:
max width is different based on first label we select, is it expected?
<img width="336" alt="Untitled 5" src="https://github.com/user-attachments/assets/dbc15a0a-6178-4de2-ad19-807535fd3ac2">
<img width="331" alt="Untitled 4" src="https://github.com/user-attachments/assets/92f90ca2-899e-43a8-8862-3f91d3dd652e">

Reply:
Yes. It will only show the first 2 labels, with a width capped at 230px. If your first 2 labels are very short, it will have different length.

We could also force the filter to have a fixed width of 230px